test-data/unit/stubgen.test

```diff
@@ -1248,6 +1262,24 @@ class F:
 @t.coroutine
 def g(): ...
 
+[case testCoroutineSpecialCase_import]
+import asyncio
+
+__all__ = ['C']
+
+@asyncio.coroutine
+def f():
```

Comment:
Why don't we include `f()` in the output?

Reply:
It's not included in `__all__`.

Reply:
(This is something we may want to change, but it may require the generation of an explicit `__all__`.)